cms/admin/utils.py

```diff
@@ -696,6 +699,51 @@ def save_model(self, request: HttpRequest, obj: models.Model, form: forms.Form,
             setattr(form._content_instance, self.grouper_field_name, obj)
             form._content_instance.save()
 
+    def get_search_fields(self, request):
+        """Return search fields for either grouper model or content model"""
+        content_search_fields = []
+        grouper_search_fields = []
+        for field_name in self.search_fields:
+            if field_name.startswith(f"{self.content_related_field}__") or field_name.startswith("content__"):
```

Comment:
```suggestion
            if field_name.startswith(CONTENT_PREFIX):
```
For syntactical consistency, I would like to see the `CONTENT_PREFIX` here. This way all admin configurations behave the same.